### PR TITLE
[cypress][SIMS #1521] Fixing Institution Test Cases - I 

### PR DESCRIPTION
**Pull request by velang01**
Fixing the test cases to match the cy-ids and some modification to the methods

### CODE REVIEW

File: testing/src/cypress/page-objects/Institution-objects/ManageUsersObject.ts [hunk 1]
```diff
@@ -35,12 +35,16 @@ export default class ManageUsersObject {
...
+    return this.addNewUserModal().contains("Add new user");
+  }
+
   isAdminRadioButton() {
     return cy.get("[data-cy='isAdmin']");
   }
 
   isLegalSigningAuthorityButton() {
-    return cy.get("[data-cy='isLegalSigningAuthority]");
+    return cy.get("[data-cy='isLegalSigningAuthority']");
```
Comment (andrewsignori-aot) [inline testing/src/cypress/page-objects/Institution-objects/ManageUsersObject.ts:47]: In the future we can consider having a help method to stop writing `"[data-cy='something']"` every time, as below:
```ts
return cy.get(dataCY("isLegalSigningAuthority"));
```

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

**Review by dheepak-aot — APPROVED** [review]
LGTM @velang01

**Review by ann-aot — APPROVED** [review]
👍

**Pull request merged**